include/rocksdb/advanced_options.h

```diff
@@ -912,7 +912,8 @@ struct AdvancedColumnFamilyOptions {
   //
   // Leveled: files older than `periodic_compaction_seconds` will be picked up
   //    for compaction and will be re-written to the same level as they were
-  //    before.
+  //    before if ttl >= periodic_compaction_seconds, or it will be compacted
```

Comment:
Could you also mention `level_compaction_dynamic_level_bytes` and that last level is still compacted to the same level?

Reply:
Done, thank you!

Reply:
Also mention that files in last level will be compacted to the same level? I think that will be the main purpose if a user wants to set this option.

Reply:
Sure, my bad.